### PR TITLE
Added the GK64 layout to DZ60

### PR DESCRIPTION
**Pull request by kb-elmo**
## Description

The DZ60 PCB was missing the GK64 layout. Since it's one of the more commonly used layouts for the board I thought it would make sense to add it.  
I used LAYOUT_gk64 as a preset name but I'm open for other suggestions there.

It has the 2u left shift and all 1u keys on the right side.
image

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Review by tzarc — APPROVED** [review]

File: keyboards/dz60/dz60.h [hunk 1]
```diff
@@ -708,3 +708,30 @@
     { k30,  k31,   k32,   k33,  k34,   k35,   k36,  k37,   k38,   k39,   k3a,  k3b,  k3c,   k3d,   k3e   }, \
     { k40,  k41,   KC_NO, k43,  KC_NO, KC_NO, k46,  KC_NO, KC_NO, KC_NO, k4a,  k4b,  k4c,   k4d,   k4e   }  \
 }
+
+/* LAYOUT_gk64
```
Comment (noroadsleft) [inline keyboards/dz60/dz60.h:712]: This is actually [one of QMK's Community Layouts](https://github.com/qmk/qmk_firmware/tree/8af72000851dbe2c50a66bf2b9452011cbfe835a/layouts/default/64_ansi).

```suggestion
/* LAYOUT_64_ansi
```

Reference: https://docs.qmk.fm/#/feature_layouts

File: keyboards/dz60/dz60.h [hunk 2]
```diff
@@ -708,3 +708,30 @@
...
+ * │10   │12 │13 │14 │15 │16 │17 │18 │19 │1a │1b │1c │1d │1e   │
+ * ├─────┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴─────┤
+ * │20    │22 │23 │24 │25 │26 │27 │28 │29 │2a │2b │2c │2d      │
+ * ├──────┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬───┬───┤
+ * │30     │32 │33 │34 │35 │36 │37 │38 │39 │3a │3b │3c │3d │3e │
+ * ├────┬──┴─┬─┴──┬┴───┴───┴───┴───┴───┴───┼───┼───┼───┼───┼───┤
+ * │40  │41  │43  │46                      │4a │4b │4c │4d │4e │
+ * └────┴────┴────┴────────────────────────┴───┴───┴───┴───┴───┘
+*/
+#define LAYOUT_gk64( \
```
Comment (noroadsleft) [inline keyboards/dz60/dz60.h:725]:
```suggestion
#define LAYOUT_64_ansi( \
```

File: keyboards/dz60/info.json [hunk 3]
```diff
@@ -1873,6 +1873,74 @@
                 {"x": 13, "y": 4},
                 {"x": 14, "y": 4}
             ]
+        },
+        "LAYOUT_gk64": {
```
Comment (noroadsleft) [inline keyboards/dz60/info.json:1877]:
```suggestion
        "LAYOUT_64_ansi": {
```

File: keyboards/dz60/keymaps/gk64/keymap.c [hunk 4]
```diff
@@ -0,0 +1,18 @@
+#include QMK_KEYBOARD_H
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+
+	LAYOUT_gk64(
```
Comment (noroadsleft) [inline keyboards/dz60/keymaps/gk64/keymap.c:5]:
```suggestion
	LAYOUT_64_ansi(
```

File: keyboards/dz60/keymaps/gk64/keymap.c [hunk 5]
```diff
@@ -0,0 +1,18 @@
...
+		KC_GESC, KC_1,    KC_2,    KC_3,    KC_4,    KC_5,    KC_6,    KC_7,    KC_8,    KC_9,    KC_0,    KC_MINS, KC_EQL,           KC_BSPC,
+		KC_TAB,           KC_Q,    KC_W,    KC_E,    KC_R,    KC_T,    KC_Y,    KC_U,    KC_I,    KC_O,    KC_P,    KC_LBRC, KC_RBRC, KC_BSLS,
+		KC_CAPS,          KC_A,    KC_S,    KC_D,    KC_F,    KC_G,    KC_H,    KC_J,    KC_K,    KC_L,    KC_SCLN, KC_QUOT, KC_ENT,
+		KC_LSFT,          KC_Z,    KC_X,    KC_C,    KC_V,    KC_B,    KC_N,    KC_M,    KC_COMM, KC_DOT,  KC_SLSH, KC_RSFT, KC_UP,   MO(1),
+		KC_LCTL, KC_LGUI,          KC_LALT,                   KC_SPC,                             KC_RALT, KC_RGUI, KC_LEFT, KC_DOWN, KC_RGHT),
+
+	LAYOUT_gk64(
```
Comment (noroadsleft) [inline keyboards/dz60/keymaps/gk64/keymap.c:12]:
```suggestion
	LAYOUT_64_ansi(
```

**Review by noroadsleft — CHANGES_REQUESTED** [review]

**Review by fauxpark — APPROVED** [review]

**Review by noroadsleft — APPROVED** [review]

**Comment by noroadsleft** [comment]
Thanks!

**Pull request merged**